.envrc.template

```diff
@@ -26,3 +26,8 @@ export DATABASE_HOSTNAME="localhost"
 
 # Access key for allowing PA Message API calls. Required for endpoints in the /api/pa-messages scope. Generate a unique random string for local use.
 export SCREENPLAY_API_KEY="local_api_key"
+
+# Host and access key needed to communicate with Watts. Required when previewing PA message audio.
+# For local use, set to the same value as your local instance of Watts.
+#export WATTS_URL=""
```

Comment:
The issue with that is not having a way to retrieve the API key. It might be worth requesting the staging key be added to 1Password and then a comment in this file can include details on where to find it. Until then, a default that isn't the local instance would not help with using the app.

Reply:
The staging key might be in the shared 1P already. Keep in mind that watts staging returns an empty response to all calls, to avoid polly costs. (It might be good to have it return a hard-coded but real mp3 file, so we could actually hear something (without linking with prod)

Reply:
It doesn't seem to be in any vault I have access to. Do you know if there is a separate vault for PA/ESS related stuff? If not, I can file an infra ticket to have the staging key added and will make the template change here. 

Good idea on the staging return change. I can tackle that as part of this task.

Reply:
[Here](https://app.asana.com/0/1113179098808463/1207601226251965/f) is the infra task for adding/giving our team access to the API key in 1Password. [Here](https://app.asana.com/0/1185117109217413/1207589542683798/f) is the task for changing the response of non-prod environments.